zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -25,11 +25,15 @@
       <term name="note">wie Anm.</term>
       <term name="et-al">et al.</term>
       <term name="editor" form="short">Hg.</term>
+      <term name="translator" form="verb-short">übers. von</term>
+      <term name="editortranslator" form="verb-short">hg. und übers. von</term>
```

Comment:
Okay, I read your comment below. But the first line seems then not necessary. Moreover, we don't use "editortranslator" at the moment anywhere in the style. Note this would be one person who edited and translated the text.